crashes.py

```diff
@@ -1454,6 +1455,54 @@ def generateTopCrashReport(reports, stats, totalCrashesProcessed, parameters, ip
                                                  processeddate=dateTime))
   resultFile.close()
 
+def generateTopCrasherIds(output, reports, reportLowerClientLimit):
+  # For each of the top signatures by crash volume:
+  IdsForMostCommon = 10
+  # we want to produce a list of crash ids per configuration:
+  IdsPerConfiguration = 100
+  # where configurations are paritioned on:
+  ConfigurationParameters = ["operatingsystem", "arch", "osversion", "firefoxver"]
```

Comment:
Keep in mind that remote settings allows for this feature if I'm reading it correctly: https://remote-settings.readthedocs.io/en/latest/target-filters.html 

So we may want to directly generate JEXL filters that remote settings will deal with rather than having our own level?

Reply:
Oh cool! We could use that for filtering release channel, among other things.

However in this code, those configuration parameters are used to distinguish unique configurations to ensure we don't exceed `IdsPerConfiguration`; I don't think that will extend to JEXL filters in a useful way. This is simply a means of limiting the number of ids we select because we don't _need_ more than `IdsPerConfiguration` (lest users pestered). That being said, I'm very curious what sort of response rate we will experience with the new UX. It might turn out that only 10% of people respond (which still would probably be a lot of usable reports).